test/sinks.test.js

```diff
@@ -290,16 +290,16 @@ describe('asset-pipe-sink-gcs', () => {
 
         let results = [];
 
-        for (let i = 0; i < 10; i++) {
-            let a = new Array(5);
+        for (let i = 0; i < 2; i++) {
+            let a = new Array(10);
             a = a.join(',').split(',');
```

Comment:
What sorcery is this?

Reply:
this sorcery needed to get 10 iterations otherwise you get a lengh property of 10 but only a single iteration using a loop.

Reply:
`Array.from(new Array(10))`

Reply:
better. To avoid any weirdness probably should have just run a second for loop and shoved the promises into an array before calling Promise.all

```js
for (let i = 0; i < 5; i++) {
  const promises = []
  for (let i = 0; i < 5; i++) {
    promises.push(doBundle())
  }
  const res = await Promise.all(promises);
  results = results.concat(res);
}
```